template/module/views/res_partner.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's for helping when searching files. If you have data and views, and you see files only with the name of the model, you won't know which one is the view and which one is the data.